plugins/foreman_openscap/0.5/index.md

```diff
@@ -0,0 +1,220 @@
+configuration or security baselines. Users are advised to acquire examplary baseline by installing scap-security-guide package.
+DataStream file usualy contains multiple XCCDF Profiles. Each for different security target. The content of DataStream file can be inspected by oscap tool from openscap-scanner package.
+(XCCDF = Extensible Configuration Checklist Description Format, XCCDF profile = A checklist which audit specific security target)
+
+__Compliance Policy__ is high level concept of a baseline applied to the infrastructure. Compliance policy is defined by user on web interface. Users may assign following information to the policy:
+
+* SCAP Content
+* XCCDF Profile from particular SCAP Content
+* Hostgroups that should comply with the policy
+* Schedule - the period in which the audit shall occur
+
+__ARF Report__ is XML output of single scan occurrence per single host. Asset Reporting File format is defined by SCAP 1.2 standard. Foreman plug-in stores the ARF Reports in database for later inspections.
+
+
+## 1.2 Release notes
+foreman_openscap has two versions: 0.4.x for 1.8+, 0.5.x for 1.11+
+
+Both versions' functionality is the same, the changes only affect to the way it communicates with Foreman and Smart-Proxy APIs
+
+<table class="table table-bordered table-condensed">
+  <tr>
+    <th>Foreman version</th>
+    <th>Plugin version</th>
+    <th>Proxy version</th>
+    <th>Client version</th>
+  </tr>
+  <tr>
+    <td>= 1.7</td>
+    <td>0.3.4</td>
+    <td>0.3.1</td>
+    <td>0.1.1</td>
+  </tr>
+  <tr>
+    <td>>= 1.8</td>
+    <td>0.4.1</td>
+    <td>0.4.0</td>
+    <td>0.1.1</td>
+  </tr>
+  <tr>
+    <td>>= 1.11</td>
+    <td>0.5.0</td>
+    <td>0.5.0</td>
+    <td>0.1.2</td>
+  </tr>
+</table>
+
+# 2. Installation
+
+There are a few components to install:
+
+Foreman OpenSCAP (foreman_openscap), Smart Proxy OpenSCAP (smart_proxy_openscap), foreman_scap_client and puppet-foreman_scap_client.
+
+### 2.1 Installing foreman_openscap
+
+On the Foreman server:
+
+	> yum install ruby193-rubygem-foreman_openscap
+	> service httpd reload
+
+Please refer the Foreman plugin [manual](http://theforeman.org/manuals/latest/index.html#6.1InstallaPlugin) for more information about installing Foreman plugins.
+
+### 2.2 Installing smart_proxy_openscap
+
+On the Smart-Proxy server:
+
+
+    > yum install rubygem-smart_proxy_openscap
+
+Edit ```openscap.yml``` with the appropriate settings
+
+    ---
+    :enabled: true
+    :reportsdir: /path/to/saved/arf_reports/
+    :spooldir: /path/to/arf_reports/which/failed/to/post/to/foreman
+
+
+### 2.3 Installing puppet-foreman_scap_client
+
+    puppet module install isimluk-foreman_scap_client
+
+This puppet module will automatically install foreman_scap_client (if not installed) and will configure the client's ```/etc/foreman_scap_client/config.yaml``` with parameters which are needed for the operation of foreman_scap_client.
+foreman_scap_client to run scans and upload results to the Smart Proxy.
```

Comment:
This line seems a bit out of context, was it meant to be part of the previous sentence?